backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/dataImport/C_DataImport_StepDef.java

```diff
@@ -301,4 +308,61 @@ public void store_business_partner_string_requestBody_in_context(@NonNull final
 			testContext.setRequestPayload(payload.replaceAll("null", ""));
 		}
 	}
+
+	@And("store modular contract DataImport String requestBody in context")
+	public void store_business_partner_string_requestBody_in_context11111(@NonNull final DataTable dataTable)
```

Comment:
not sure this method belongs to this class.
also, check the method name (11111)

Reply:
it kinda does, since it's related to setting data in context. fixed method name though.